clients/cli/src/analytics.rs

```diff
@@ -16,14 +16,27 @@ pub fn track(
 ) {
     println!("{}", description);
 
-    let token = analytics_token(ws_addr_string);
-    if token.is_empty() {
+    let firebase_app_id = analytics_token(ws_addr_string);
+    let firebase_api_secret = analytics_api_secret(ws_addr_string);
+    if firebase_app_id.is_empty() {
         return;
     }
     let local_now = chrono::offset::Local::now();
+
+    // For tracking events, we use the Firebase Measurement Protocol
+    // Firebase is mostly designed for mobile and web apps, but for our use case of a CLI,
+    // we can use the Measurement Protocol to track events by POST to a URL. 
+    // The only thing that may be unexpected is that the URL we use includes a firebase secret (something we dont typically add to client code)
```

Comment:
How does it work on the web version? Where does the secret come from?

Reply:
As long as this is a secret we can revoke I think it's ok to check it in, there's no way we can hide it from CLI users if the CLI is open source

Reply:
The docs say: We recommend that you keep these private to your organization. If you deploy the measurement protocol client-side, you should regularly rotate api_secrets to avoid excessive SPAM.

I think there's not much we can do, we just have to plan to rotate them.